CODEOWNERS

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This CODEOWNERS file defines individuals or teams that are responsible
+# for code in this repository. Code owners are automatically requested
+# for review when someone opens a pull request that modifies code that
+# they own. Order is important; the last matching pattern takes the most
+# precedence.
+# A CODEOWNERS file uses a pattern that follows the same rules used in
+# gitignore files. The pattern is followed by one or more GitHub usernames
+# or team names using the standard @username or @org/team-name format.
+# You can also refer to a user by an email address that has been added
+# to their GitHub account, for example user@example.com.
+# https://help.github.com/articles/about-codeowners/
+
+# Release Engineering pipeline
+/.cron.yml                      @mozilla-mobile/releng
+/.taskcluster.yml               @mozilla-mobile/releng
+/android-components/automation/ @mozilla-mobile/releng @mozilla-mobile/aet @mozilla-mobile/QA
+/fenix/automation/              @mozilla-mobile/releng @mozilla-mobile/aet @mozilla-mobile/QA
+/focus-android/automation/      @mozilla-mobile/releng @mozilla-mobile/aet @mozilla-mobile/QA
```

Comment:
@AaronMT @rpappalax I added the QA team to these directories based on what I saw in the activity git history, but I'm unsure if this the write team or if you would prefer another one.

Reply:
I think releng can be removed from those `/*/automation/` directories, FWIW.

Reply:
Sorry I missed this, @mozilla-mobile/QA is fine for now. There's a bunch of @mozilla-mobile team maintenance we should look at cleaning up and adding people. We may adjust this to @mozilla-mobile/mobile-ci